module4/labs/Week2/BuildAnAPI.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Build an API - Lab
+---
+
+## Lab Objectives
+* Practice building a RESTful CRUD API
+* Use our knowledge of MVC and APIs to add a related resource
+
+Today, we will be working individually to reinforce our own understanding of the components of an API.  You _should_ reach out to other students and instructors with questions - we can all help each other learn!
+
+By the end of this Lab, you will have a RESTful API that will allow us to CRUD resources for a database with the following ERD:
+
+![Adopt Don't Shop ERD](/assets/images/module4/week2/adoptdontshop_ERD.png)
+
+## Setup
+
+* Fork and clone this [Starter Repo]().
```

Comment:
Any progress on the starter repo yet, or will this be built from a previous lesson's lab?

Reply:
whoops!  It is now linked!